utils/db.js

```diff
@@ -117,4 +117,18 @@ const updateUserTag = async (userAddress, accessToken, newTag) => {
     return await response;
 };
 
-module.exports = { getMessages, storeMessages, getUser, getUsers, updateUser, getDiscordSettings, db, updateUserTag };
+const addNewOrganization = async (organizationName, address, image) => {
+    let dbParams = {
+        TableName: "Organizations",
+        Item: {
+            identifier: address,
+            organizationId: address,
+            image: image,
+            name: organizationName,
```

Comment:
Create a new entry called `createdBy: address` will be helpful to track who created it if we change the identifier tomorrow.